src/axom/inlet/tests/inlet_writer.cpp

```diff
@@ -0,0 +1,253 @@
+// Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
```

Comment:
Minor: Should this file be named something like `inlet_jsonschema` instead of `inlet_writer` ?